### PR TITLE
Add support to properties()

### PR DESCRIPTION
**Pull request by nafraf**
Hi, this PR implements the function properties() #2615
Changes done:
- [x] Implement properties() as part of map_functions
- [x] Add tests to test_function_calls.py
- [x] Remove @skip labes in Graph10.feature
- [x] Add properties() to docs

Comments are welcome. Thanks.

### CODE REVIEW

File: src/arithmetic/map_funcs/map_funcs.c [hunk 1]
```diff
@@ -57,6 +58,29 @@ SIValue AR_KEYS(SIValue *argv, int argc, void *private_data) {
...
 }
 
+SIValue AR_PROPERTIES(SIValue *argv, int argc, void *private_data) {
+	ASSERT(argc == 1);
+
+	if(SI_TYPE(argv[0]) == T_NULL) {
+		return SI_NullVal();
+	} else if(SI_TYPE(argv[0]) == T_MAP) {
+		return argv[0];
+	} else if(SI_TYPE(argv[0]) & (T_MAP | SI_GRAPHENTITY)) {
```
Comment (AviAvni) [inline src/arithmetic/map_funcs/map_funcs.c:68]: is T_MAP is redundant?

File: src/graph/entities/graph_entity.c [hunk 2]
```diff
@@ -73,6 +73,26 @@ SIValue GraphEntity_Keys
...
+// returns an SIArray of all keys and values in graph entity properties. 
+// Keys at even positions, Values at odd position of the array
+SIValue GraphEntity_Properties
+(
+	const GraphEntity *e
+) {
+	GraphContext *gc = QueryCtx_GetGraphCtx();
+	const AttributeSet set = GraphEntity_GetAttributes(e);
+	int propCount = ATTRIBUTE_SET_COUNT(set);
+	SIValue properties = SI_Array(propCount);
```
Comment (AviAvni) [inline src/graph/entities/graph_entity.c:85]: why this is not return map?

File: src/arithmetic/map_funcs/map_funcs.c (hunk 1 above)
Comment (nafraf) [reply src/arithmetic/map_funcs/map_funcs.c]: Fixed in [2e22b7b](https://github.com/RedisGraph/RedisGraph/pull/2632/commits/2e22b7b92699fc7027530007f0047022fdca9018)

File: src/graph/entities/graph_entity.c (hunk 2 above)
Comment (nafraf) [reply src/graph/entities/graph_entity.c]: You are right, it is better to return map. Change done in [2e22b7b](https://github.com/RedisGraph/RedisGraph/pull/2632/commits/2e22b7b92699fc7027530007f0047022fdca9018)

File: src/arithmetic/map_funcs/map_funcs.c (hunk 1 above)
Comment (nafraf) [reply src/arithmetic/map_funcs/map_funcs.c]: Fixed in [0cc2dbb](https://github.com/RedisGraph/RedisGraph/pull/2632/commits/0cc2dbb4e25954c4ea584931f74c2f50b0c11f28). The if/else was replaced by switch/case

**Pull request merged**